packages/ui-react/src/containers/StartWatchingButton/StartWatchingButton.tsx

```diff
@@ -70,6 +73,11 @@ const StartWatchingButton: React.VFC<Props> = ({ item, playUrl, disabled = false
     return () => setRequestedMediaOffers(null);
   }, [mediaOffers, setRequestedMediaOffers]);
 
+  // the user can't purchase access in an AVOD platform due to missing configuration, so we hide the button
+  if (accessModel === ACCESS_MODEL.AVOD && !isEntitled) {
```

Comment:
No, because we still want to show the button for AVOD (free) content. `isEntitled` is set to `true` when the user can watch the item for all access models.